tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -217,6 +217,8 @@ def create_parser(product_choices=None):
                               help="Run browser in headless mode", default=None)
     config_group.add_argument("--no-headless", action="store_false", dest="headless",
                               help="Don't run browser in headless mode")
+    config_group.add_argument("--instruments", action="store",
```

Comment:
Should this be singular (`--instrument`) or more explicit (`--instrument-to-file`)? I'm reading it as an active verb, maybe you're meaning it to be read differently.